pkg/apis/awsproviderconfig/v1beta1/awsmachineproviderconfig_types.go

```diff
@@ -60,6 +60,16 @@ const (
 	MachineCreation AWSMachineProviderConditionType = "MachineCreation"
 )
 
+// AWSMachineProviderConditionReason is reason for the condition's last transition
```

Comment:
Nit: should have period at end
```suggestion
// AWSMachineProviderConditionReason is reason for the condition's last transition.
```